doc/getting-started/getting-started.rst

```diff
@@ -579,9 +582,9 @@ Build ACRN
       .. code-block:: bash
 
          disk="/media/$USER/"$(ls /media/$USER)
-         cp ~/acrn-work/acrn-hypervisor/build/acrn-my_board-shared-2.7.deb "$disk"/ # TODO update file name
+         cp ~/acrn-work/acrn-hypervisor/build/acrn-my_board-MyConfiguration-3.0-acrn-*.deb "$disk"/ # TODO update file name
```

Comment:
If that's indeed the right file name now, remove the #TODO comment

Reply:
update